guides/common/modules/proc_assigning-resource-quotas-to-a-user-group.adoc

```diff
@@ -3,8 +3,34 @@
 
 You can assign resource quotas to a user group to limit the resource consumption of that {Project} user group.
 All users of the user group share the pool of resources.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-assigning-resource-quotas-to-a-user-group[].
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Administer* > *User Groups*.
 . On the *Resource Quota* tab, assign resource quotas to the user group.
 . Click *Submit* to assign resource quotas to a user group.
+
+[id="cli-assigning-resource-quotas-to-a-user-group"]
+.CLI procedure
+. Optional: List all {Project} user groups:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer user-group list --fields id,name
+----
+. Optional: List all resource quotas:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer resource-quota list --fields id,name
+----
+. Assign resource quotas to a user group:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer user-group update \
+--id _My_User_Group_ID_ \
+--resource-quota-ids _My_Resource_Quota_IDs_
+----
++
+For all options, see `hammer user-group --help`.
```

Comment:
Same as for users. You will only see a resource quota option for create/update actions.

Reply:
Added "update".